### PR TITLE
debug code for bootstrap-tab-history-custom.js

### PR DESCRIPTION
**Pull request by orangejenny**
@proteusvacuum I started seeing the blank screen issue you've been getting, but only intermittently on india, and I haven't had much luck working in the chrome debugger to figure out what's going on. Adding some temporary debug code to narrow down where the problem is.

### CODE REVIEW

File: corehq/apps/hqwebapp/static/hqwebapp/js/lib/bootstrap-tab-history-custom.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ $(function () {
                 State.data.tab = '';
             }
         }
+        console.log("State.data.tab = " + State.data.tab);
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/static/hqwebapp/js/lib/bootstrap-tab-history-custom.js]: Unexpected console statement. (no-console)

File: corehq/apps/hqwebapp/static/hqwebapp/js/lib/bootstrap-tab-history-custom.js [hunk 2]
```diff
@@ -51,6 +52,11 @@ $(function () {
                 }, null, State.url);
             }
         }
+        if (link.length) {
+            console.log(link.get(0).outerHTML);
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/static/hqwebapp/js/lib/bootstrap-tab-history-custom.js]: Unexpected console statement. (no-console)

File: corehq/apps/hqwebapp/static/hqwebapp/js/lib/bootstrap-tab-history-custom.js [hunk 3]
```diff
@@ -51,6 +52,11 @@ $(function () {
...
+        } else {
+            console.log("No link found");
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/static/hqwebapp/js/lib/bootstrap-tab-history-custom.js]: Unexpected console statement. (no-console)

**Comment by orangejenny** [comment]
no test coverage, killed build

**Pull request merged**